geomstats/_backend/tensorflow/__init__.py

```diff
@@ -791,6 +807,19 @@ def where(condition, x=None, y=None):
     return tf.where(condition, x, y)
 
 
+def tril_to_vec(x, k=0):
```

Comment:
Can we write this like in Numpy or pytorch using `tril_indices`? If so, this function should also go to algebra_utils.py